### ISSUE TITLE
Add_Photos not adding photos prior to start

### ISSUE THREAD

**Issue by Blobbyguy** [fastlane/fastlane#21607]
### New Issue Checklist

- [Y] Updated fastlane to the latest version
- [Y] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [Y] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [Y] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

The add_photos argument is failing to add the photos prior to the simulator starting. The simulator therefore, does not recognise the photos are present, and hence are not registering in the photos app and cannot be used during the simulation.

Removing the *erase simulator* and running for a 2nd time will **sometimes** produce the correct output, confirming that the file is being accessed correctly, however, it is quite inconvenient. 

This doesn't feel like expected behaviour to me - and I can't find any documentation regarding this - obviously call me out if I'm doing something incorrect.

Furthermore - it may be a sign of something more wrong - the html page that gets created is empty, but the screenshots are taken successfully. See the below to see that the photos do not get added (I took a snapshot to see what was actually present) 

![iPhone 15-Image Options](https://github.com/fastlane/fastlane/assets/20289715/5af6a0c8-a22d-4cff-8a68-f48876f04dcc)

##### Command executed

  snapshot(
    headless: true,
    add_photos: media_file_path,
    clean: true, # Clean the project before running
    devices: simulator_udids, #Stuff
    languages: 'en-US', # Specify the language/locale if needed
    clear_previous_screenshots: true,
    erase_simulator: true,
    override_status_bar: true,
    number_of_retries: 0,
    stop_after_first_error: true,
  )

##### Complete output when running fastlane, including the stack trace and command used

**Note; I've also omitted some logs relating to https://developer.apple.com/forums/thread/737923 to fit the character count of github**

<details>
<pre> 
[09:37:21]: [33mGet started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile[0m
[09:37:21]: [32mDriving the lane 'run_ui_test_and_capture_screenshots' 🚀[0m
[09:37:22]: [32m----------------------[0m
[09:37:22]: [32m--- Step: snapshot ---[0m
[09:37:22]: [32m----------------------[0m
[09:37:22]: [32mSuccessfully loaded '/Users/sam/Desktop/XCode Applications/REDACTED/fastlane/Snapfile' 📄[0m

+----------------------------------------------------------------+
|           Detected Values from './fastlane/Snapfile'           |
+----------------------------+-----------------------------------+
| devices                    | ["iPhone 15"]                     |
| output_directory           | ./screenshots                     |
| clear_previous_screenshots | true                              |
| override_status_bar        | true                              |
| add_photos                 | ./REDACTEDUITests/DemoImage.png |
| scheme                     | REDACTED                        |
| clean                      | true                              |
| reinstall_app              | true                              |
| number_of_retries          | 0                                 |
| stop_after_first_error     | true                              |
| headless                   | true                              |
+----------------------------+-----------------------------------+

[09:37:22]: [33mResolving Swift Package Manager dependencies...[0m
[09:37:22]: [36m$ xcodebuild -resolvePackageDependencies -scheme REDACTED -project ./REDACTED.xcodeproj[0m
[09:37:23]: ▸ [35mCommand line invocation:[0m
[09:37:23]: ▸ [35m    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -resolvePackageDependencies -scheme REDACTED -project ./REDACTED.xcodeproj[0m
[09:37:23]: ▸ [35mUser defaults from command line:[0m
[09:37:23]: ▸ [35m    IDEPackageSupportUseBuiltinSCM = YES[0m
[09:37:23]: ▸ [35mresolved source packages: [0m
[09:37:23]: [36m$ xcodebuild -showBuildSettings -scheme REDACTED -project ./REDACTED.xcodeproj[0m

+------------------------------------------------------------------------------+
|                         [32mSummary for snapshot 2.216.0[0m                         |
+--------------------------------------+---------------------------------------+
| headless                             | true                                  |
| app_identifier                       | com.Blobbyguy.REDACTED              |
| add_photos                           | ["./REDACTEDUITests/DemoImage.png"] |
| clean                                | true                                  |
| devices                              | ["iPhone 15"]                         |
| languages                            | ["en-US"]                             |
| clear_previous_screenshots           | true                                  |
| erase_simulator                      | true                                  |
| override_status_bar                  | true                                  |
| number_of_retries                    | 0                                     |
| stop_after_first_error               | true                                  |
| project                              | ./REDACTED.xcodeproj                |
| launch_arguments                     | [""]                                  |
| output_directory                     | /Users/sam/Desktop/XCode              |
|                                      | Applications/REDACTED/screenshots   |
| output_simulator_logs                | false                                 |
| skip_open_summary                    | false                                 |
| skip_helper_version_check            | false                                 |
| reinstall_app                        | true                                  |
| localize_simulator                   | false                                 |
| buildlog_path                        | ~/Library/Logs/snapshot               |
| scheme                               | REDACTED                            |
| result_bundle                        | false                                 |
| concurrent_simulators                | true                                  |
| disable_slide_to_type                | false                                 |
| skip_package_dependencies_resolution | false                                 |
| disable_package_automatic_updates    | false                                 |
| xcodebuild_formatter                 | xcpretty                              |
| use_system_scm                       | false                                 |
| xcode_path                           | /Applications/Xcode.app               |
+--------------------------------------+---------------------------------------+

[09:37:24]: [33mClearing previously generated screenshots[0m
[09:37:24]: [32mBuilding and running project - this might take some time...[0m
[09:37:24]: Patching simulators '/Users/sam/Library/Preferences/com.apple.iphonesimulator.plist' to scale to 100%
[09:37:25]: [33mErasing iPhone 15...[0m
[09:37:25]: Launch Simulator iPhone 15
[09:37:25]: [36m$ open -a Simulator.app --args -CurrentDeviceUDID 19CD1030-E602-4378-AADB-185D8052DDE0 &> /dev/null[0m
[09:37:25]: Adding './REDACTEDUITests/DemoImage.png'
[09:37:25]: [36m$ xcrun simctl addmedia 19CD1030-E602-4378-AADB-185D8052DDE0 ./REDACTEDUITests/DemoImage.png &> /dev/null[0m
[09:37:26]: [33mUsing device named 'iPhone 15' with version '17.0.1' because no match was found for version '17.0'[0m
[09:37:26]: Launch Simulator iPhone 15
[09:37:26]: [36m$ xcrun simctl bootstatus 19CD1030-E602-4378-AADB-185D8052DDE0 -b &> /dev/null[0m
[09:37:28]: Waiting 0 seconds for device to fully boot before overriding status bar... Set 'SNAPSHOT_SIMULATOR_WAIT_FOR_BOOT_TIMEOUT' environment variable to adjust timeout
[09:37:28]: Overriding Status Bar
[09:37:28]: [36m$ xcrun simctl status_bar 19CD1030-E602-4378-AADB-185D8052DDE0 override --time 2007-01-09T09:41:00+11:00 --dataNetwork wifi --wifiMode active --wifiBars 3 --cellularMode active --operatorName '' --cellularBars 4 --batteryState charged --batteryLevel 100 &> /dev/null[0m
[09:37:28]: [33mRunning snapshot on: iPhone 15[0m
[09:37:28]: [36m$ set -o pipefail && xcodebuild -scheme REDACTED -project ./REDACTED.xcodeproj -derivedDataPath /var/folders/yp/mxp8v99n09j0y9j_rkc0jjq80000gn/T/snapshot_derived20231101-91171-wqrtdg -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.0.1' FASTLANE_SNAPSHOT=YES FASTLANE_LANGUAGE=en-US clean build test | tee /Users/sam/Library/Logs/snapshot/REDACTED-REDACTED.log | xcpretty [0m
[09:37:28]: ▸ [35mLoading...[0m
[09:37:29]: ▸ [35m--- xcodebuild: WARNING: Using the first of multiple matching destinations:[0m
[09:37:29]: ▸ [35m{ platform:iOS Simulator, id:19CD1030-E602-4378-AADB-185D8052DDE0, OS:17.0.1, name:iPhone 15 }[0m
[09:37:29]: ▸ [35m{ platform:iOS Simulator, id:19CD1030-E602-4378-AADB-185D8052DDE0, OS:17.0.1, name:iPhone 15 }[0m
[09:37:29]: [35m[33m▸[0m [39;1mClean[0m Succeeded[0m
[09:37:29]: ▸ [35m--- xcodebuild: WARNING: Using the first of multiple matching destinations:[0m
[09:37:29]: ▸ [35m{ platform:iOS Simulator, id:19CD1030-E602-4378-AADB-185D8052DDE0, OS:17.0.1, name:iPhone 15 }[0m
[09:37:29]: ▸ [35m{ platform:iOS Simulator, id:19CD1030-E602-4378-AADB-185D8052DDE0, OS:17.0.1, name:iPhone 15 }[0m
[09:37:29]: ▸ [35m2023-11-01 09:37:29.934 xcodebuild[91453:2503026] [MT] DVTAssertions: Warning in /System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot11/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/IDEFrameworks/IDEFrameworks-22269/IDEFoundation/Provisioning/Capabilities Infrastructure/IDECapabilityQuerySelection.swift:103[0m
[09:37:29]: ▸ [35mDetails:  createItemModels creation requirements should not create capability item model for a capability item model that already exists.[0m
[09:37:29]: ▸ [35mFunction: createItemModels(for:itemModelSource:)[0m
[09:37:29]: ▸ [35mThread:   <_NSMainThread: 0x6000033642c0>{number = 1, name = main}[0m
[09:37:29]: ▸ [35mPlease file a bug at https://feedbackassistant.apple.com with this warning message and any useful information you can provide.[0m

[09:37:29]: ▸ [35mFunction: createItemModels(for:itemModelSource:)[0m
[09:37:29]: ▸ [35mThread:   <_NSMainThread: 0x6000033642c0>{number = 1, name = main}[0m
[09:37:29]: ▸ [35mPlease file a bug at https://feedbackassistant.apple.com with this warning message and any useful information you can provide.[0m
[09:37:29]: ▸ [35m2023-11-01 09:37:29.938 xcodebuild[91453:2503026] [MT] DVTAssertions: Warning in /System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot11/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/IDEFrameworks/IDEFrameworks-22269/IDEFoundation/Provisioning/Capabilities Infrastructure/IDECapabilityQuerySelection.swift:103[0m
[09:37:29]: ▸ [35mDetails:  createItemModels creation requirements should not create capability item model for a capability item model that already exists.[0m
[09:37:29]: ▸ [35mFunction: createItemModels(for:itemModelSource:)[0m
[09:37:29]: ▸ [35mThread:   <_NSMainThread: 0x6000033642c0>{number = 1, name = main}[0m
[09:37:29]: ▸ [35mPlease file a bug at https://feedbackassistant.apple.com with this warning message and any useful information you can provide.[0m
[09:37:31]: ▸ [35m2023-11-01 09:37:31.111 xcodebuild[91453:2503064]  DVTPortal: Service '<DVTPortalViewDeveloperService: 0x600003c80d00; action='viewDeveloper'>' encountered an unexpected result code from the portal ('1100')[0m
[09:37:31]: ▸ [35m2023-11-01 09:37:31.111 xcodebuild[91453:2503064]  DVTPortal: Error:[0m
[09:37:31]: ▸ [35mError Domain=DVTPortalServiceErrorDomain Code=1100 "Your session has expired. Please log in." UserInfo={payload={[0m
[09:37:31]: ▸ [35m    creationTimestamp = "2023-10-31T22:37:30Z";[0m
[09:37:31]: ▸ [35m    httpCode = 200;[0m
[09:37:31]: ▸ [35m    protocolVersion = QH65B2;[0m
[09:37:31]: ▸ [35m    requestUrl = "https://developerservices2.apple.com/services/QH65B2/viewDeveloper.action";[0m
[09:37:31]: ▸ [35m    responseId = "7c038419-9d91-44d8-90c9-5e0c0c55d610";[0m
[09:37:31]: ▸ [35m    resultCode = 1100;[0m
[09:37:31]: ▸ [35m    resultString = "Your session has expired. Please log in.";[0m
[09:37:31]: ▸ [35m    userLocale = "en_US";[0m
[09:37:31]: ▸ [35m    userString = "Your session has expired. Please log in.";[0m
[09:37:31]: ▸ [35m}, NSLocalizedDescription=Your session has expired. Please log in.}[0m
[09:37:31]: ▸ [35m2023-11-01 09:37:31.633 xcodebuild[91453:2503064]  DVTPortal: Service '<DVTPortalViewDeveloperService: 0x600003cb2080; action='viewDeveloper'>' encountered an unexpected result code from the portal ('1100')[0m
[09:37:31]: ▸ [35m2023-11-01 09:37:31.634 xcodebuild[91453:2503064]  DVTPortal: Error:[0m
[09:37:31]: ▸ [35mError Domain=DVTPortalServiceErrorDomain Code=1100 "Your session has expired. Please log in." UserInfo={payload={[0m
[09:37:31]: ▸ [35m    creationTimestamp = "2023-10-31T22:37:31Z";[0m
[09:37:31]: ▸ [35m    httpCode = 200;[0m
[09:37:31]: ▸ [35m    protocolVersion = QH65B2;[0m
[09:37:31]: ▸ [35m    requestUrl = "https://developerservices2.apple.com/services/QH65B2/viewDeveloper.action";[0m
[09:37:31]: ▸ [35m    responseId = "8342e4dc-6d7f-4c9c-a66f-ee76d492ed18";[0m
[09:37:31]: ▸ [35m    resultCode = 1100;[0m
[09:37:31]: ▸ [35m    resultString = "Your session has expired. Please log in.";[0m
[09:37:31]: ▸ [35m    userLocale = "en_US";[0m
[09:37:31]: ▸ [35m    userString = "Your session has expired. Please log in.";[0m
[09:37:31]: ▸ [35m}, NSLocalizedDescription=Your session has expired. Please log in.}[0m
[09:37:39]: [35m[33m▸[0m [39;1mProcessing[0m Info.plist[0m
[09:37:39]: [35m[33m▸[0m [39;1mLinking[0m REDACTED[0m
[09:37:39]: [35mRunning Tests: [33m▸[0m [39;1mTouching[0m REDACTED.app (in target 'REDACTED' from project 'REDACTED')[0m
[09:37:39]: [35m[33m▸[0m [39;1mBuild[0m Succeeded[0m
[09:37:39]: ▸ [35m--- xcodebuild: WARNING: Using the first of multiple matching destinations:[0m
[09:37:39]: ▸ [35m{ platform:iOS Simulator, id:19CD1030-E602-4378-AADB-185D8052DDE0, OS:17.0.1, name:iPhone 15 }[0m
[09:37:39]: ▸ [35m{ platform:iOS Simulator, id:19CD1030-E602-4378-AADB-185D8052DDE0, OS:17.0.1, name:iPhone 15 }[0m
[09:37:39]: [35m[33m▸[0m [39;1mProcessing[0m Info.plist[0m
[09:37:39]: [35m[33m▸[0m [39;1mLinking[0m REDACTED[0m
[09:37:42]: [35m[33m▸[0m [39;1mProcessing[0m empty-REDACTEDTests.plist[0m
[09:37:42]: [35m[33m▸[0m [39;1mLinking[0m REDACTEDTests[0m
[09:37:42]: [35mRunning Tests: [33m▸[0m [39;1mTouching[0m REDACTEDTests.xctest (in target 'REDACTEDTests' from project 'REDACTED')[0m
[09:37:43]: [35m[33m▸[0m [39;1mProcessing[0m empty-REDACTEDUITests.plist[0m
[09:37:43]: [35m[33m▸[0m [39;1mLinking[0m REDACTEDUITests[0m
[09:37:43]: [35m[33m▸[0m [39;1mCopying[0m Info.plist[0m
[09:37:43]: [35mRunning Tests: [33m▸[0m [39;1mTouching[0m REDACTEDUITests.xctest (in target 'REDACTEDUITests' from project 'REDACTED')[0m
[09:38:55]: ▸ [35m[39;1mAll tests[0m
[09:38:55]: ▸ [35mTest Suite [39;1mREDACTEDTests.xctest[0m started[0m
[09:38:55]: ▸ [35m[39;1mImageSplitTest[0m
[09:38:55]: ▸ [35m    [33m◷[0m testAppendBottom measured (0.000 seconds)[0m
[09:38:55]: ▸ [35m    [32;1m✓[0m testAppendBottom ([31m0.330[0m seconds)[0m
[09:38:56]: ▸ [35m    [33m◷[0m testAppendRight measured (0.000 seconds)[0m
[09:38:56]: ▸ [35m    [32;1m✓[0m testAppendRight ([31m0.259[0m seconds)[0m
[09:38:56]: ▸ [35m    [32;1m✓[0m testSimpleSplitQuad ([33m0.045[0m seconds)[0m
[09:38:56]: ▸ [35m[32;1m	 Executed 3 tests, with 0 failures (0 unexpected) in 0.633 (0.634) seconds[0m
[09:38:56]: ▸ [35m[0m
[09:38:58]: ▸ [35m[39;1mSelected tests[0m
[09:38:58]: ▸ [35mTest Suite [39;1mREDACTEDUITests.xctest[0m started[0m
[09:38:58]: ▸ [35m[39;1mREDACTEDUITests[0m
[09:39:24]: ▸ [35m    [32;1m✓[0m testtakeScreenshots ([31m26.472[0m seconds)[0m
[09:39:24]: ▸ [35m[32;1m	 Executed 1 test, with 0 failures (0 unexpected) in 26.472 (26.473) seconds[0m
[09:39:24]: ▸ [35m[0m
[09:39:24]: ▸ [35m2023-11-01 09:39:24.917 xcodebuild[91453:2503026] [MT] IDETestOperationsObserverDebug: 101.498 elapsed -- Testing started completed.[0m
[09:39:24]: ▸ [35m2023-11-01 09:39:24.917 xcodebuild[91453:2503026] [MT] IDETestOperationsObserverDebug: 0.000 sec, +0.000 sec -- start[0m
[09:39:24]: ▸ [35m2023-11-01 09:39:24.917 xcodebuild[91453:2503026] [MT] IDETestOperationsObserverDebug: 101.498 sec, +101.498 sec -- end[0m
[09:39:25]: [35m[33m▸[0m [39;1mTest[0m Succeeded[0m
[09:39:28]: Clearing Status Bar Override
[09:39:28]: [36m$ xcrun simctl status_bar 19CD1030-E602-4378-AADB-185D8052DDE0 clear &> /dev/null[0m
[09:39:28]: [32mCopying '/Users/sam/Desktop/XCode Applications/REDACTED/screenshots/en-US/iPhone 15-Base Snapshot.png'...[0m
[09:39:28]: [32mCopying '/Users/sam/Desktop/XCode Applications/REDACTED/screenshots/en-US/iPhone 15-Image Options.png'...[0m
[09:39:28]: [32mCopying '/Users/sam/Desktop/XCode Applications/REDACTED/screenshots/en-US/iPhone 15-Image Preview.png'...[0m
[09:39:28]: [32mCopying '/Users/sam/Desktop/XCode Applications/REDACTED/screenshots/en-US/iPhone 15-Image Selected.png'...[0m
[09:39:28]: [32mCopying '/Users/sam/Desktop/XCode Applications/REDACTED/screenshots/en-US/iPhone 15-Image Tiled.png'...[0m

+-------------------+
| snapshot results  |
+-----------+-------+
| Device    | en-US |
+-----------+-------+
| iPhone 15 |  💚   |
+-----------+-------+

[09:39:28]: Generating HTML Report
[09:39:28]: [32mSuccessfully created HTML file with an overview of all the screenshots: '/Users/sam/Desktop/XCode Applications/REDACTED/screenshots/screenshots.html'[0m

+-------------------------------+
|       [32mfastlane summary[0m        |
+------+----------+-------------+
| Step | Action   | Time (in s) |
+------+----------+-------------+
| 1    | snapshot | 126         |
+------+----------+-------------+

[09:39:29]: [32mfastlane.tools finished successfully 🎉[0m

</pre>
</details>

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                      |
| --------------------------- | ---------------------------------------------------------- |
| OS                          | 14.1                                                       |
| Ruby                        | 3.1.4                                                      |
| Bundler?                    | false                                                      |
| Git                         | git version 2.41.0                                         |
| Installation Source         | /opt/homebrew/Cellar/fastlane/2.216.0/libexec/bin/fastlane |
| Host                        | macOS 14.1 (23B74)                                         |
| Ruby Lib Dir                | /opt/homebrew/Cellar/ruby@3.1/3.1.4/lib                    |
| OpenSSL Version             | OpenSSL 3.1.0 14 Mar 2023                                  |
| Is contained                | false                                                      |
| Is homebrew                 | true                                                       |
| Is installed via Fabric.app | false                                                      |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                |
| Xcode Version               | 15.0.1                                                     |
| Swift Version               | 5.9                                                        |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_AU.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
lane :run_ui_test_and_capture_screenshots do
  # Specify the simulator UDID (e.g., iPhone 11 Pro Max)
  simulator_udids = [
    #"iPhone 14 Plus",
   	"iPhone 15",
   	# "iPhone 15 Plus",
    # "iPad Pro (12.9-inch) (2nd generation)",
    # "iPad Pro (12.9-inch) (6th generation)",
    # "iPhone SE (3rd generation)"
    ]


  # Specify the path to your media file (e.g., MyFile.jpg)
  media_file_path = './REDACTED/DemoImage.png'

  # Configure snapshot to run UI tests
  snapshot(
    headless: true,
    add_photos: media_file_path,
    clean: true,
    devices: simulator_udids, 
    languages: 'en-US', 
    clear_previous_screenshots: true,
    erase_simulator: true,
    override_status_bar: true,
    number_of_retries: 0,
    stop_after_first_error: true,
  )

  # # Add media to the simulator
  # simulator_udids.each do |udid|
  #   # sh "xcrun simctl addmedia \"#{udid}\" #{media_file_path}"
  #   sh "xcrun simctl addmedia \"#{udid}\" \"#{media_file_path}\""
  # end


end

```
</details>

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.216.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| error_highlight                 | 0.3.0        |
| did_you_mean                    | 1.6.1        |
| public_suffix                   | 5.0.3        |
| addressable                     | 2.8.5        |
| artifactory                     | 3.0.15       |
| jmespath                        | 1.6.2        |
| aws-partitions                  | 1.827.0      |
| aws-eventstream                 | 1.2.0        |
| aws-sigv4                       | 1.6.0        |
| aws-sdk-core                    | 3.183.0      |
| aws-sdk-kms                     | 1.71.0       |
| aws-sdk-s3                      | 1.135.0      |
| babosa                          | 1.0.4        |
| bundler                         | 2.4.10       |
| rexml                           | 3.2.6        |
| CFPropertyList                  | 3.0.6        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| dotenv                          | 2.8.1        |
| emoji_regex                     | 3.2.3        |
| excon                           | 0.103.0      |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| multipart-post                  | 2.3.0        |
| faraday-multipart               | 1.0.4        |
| faraday-net_http                | 1.0.1        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| faraday-retry                   | 1.0.3        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.10.3       |
| faraday_middleware              | 1.2.0        |
| unf_ext                         | 0.0.8.2      |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| http-cookie                     | 1.0.5        |
| faraday-cookie_jar              | 0.0.7        |
| fastimage                       | 2.2.7        |
| gh_inspector                    | 1.1.3        |
| uber                            | 0.1.0        |
| declarative                     | 0.0.20       |
| trailblazer-option              | 0.1.2        |
| representable                   | 3.2.0        |
| retriable                       | 3.1.2        |
| mini_mime                       | 1.1.5        |
| jwt                             | 2.7.1        |
| multi_json                      | 1.15.0       |
| os                              | 1.1.4        |
| signet                          | 0.18.0       |
| googleauth                      | 1.8.1        |
| httpclient                      | 2.8.3        |
| webrick                         | 1.8.1        |
| google-apis-core                | 0.11.1       |
| google-apis-androidpublisher_v3 | 0.50.0       |
| google-apis-playcustomapp_v1    | 0.13.0       |
| google-cloud-env                | 1.6.0        |
| google-cloud-errors             | 1.3.1        |
| google-cloud-core               | 1.6.0        |
| google-apis-iamcredentials_v1   | 0.17.0       |
| google-apis-storage_v1          | 0.19.0       |
| rake                            | 13.0.6       |
| digest-crc                      | 0.6.5        |
| google-cloud-storage            | 1.44.0       |
| json                            | 2.6.1        |
| mini_magick                     | 4.12.0       |
| naturally                       | 2.2.1        |
| optparse                        | 0.1.1        |
| plist                           | 3.7.0        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.3        |
| simctl                          | 1.6.10       |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 2.4.2        |
| terminal-table                  | 3.0.2        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| word_wrap                       | 1.0.0        |
| atomos                          | 0.1.3        |
| claide                          | 1.1.0        |
| colored2                        | 3.1.2        |
| nanaimo                         | 0.3.0        |
| xcodeproj                       | 1.23.0       |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| xcpretty-travis-formatter       | 1.0.1        |
| set                             | 1.0.2        |
| forwardable                     | 1.3.2        |
| logger                          | 1.5.0        |
| pathname                        | 0.2.0        |
| shellwords                      | 0.1.0        |
| cgi                             | 0.3.6        |
| date                            | 3.2.2        |
| timeout                         | 0.2.0        |
| stringio                        | 3.0.1        |
| securerandom                    | 0.2.0        |
| uri                             | 0.12.1       |
| openssl                         | 3.0.1        |
| digest                          | 3.1.0        |
| io-nonblock                     | 0.1.0        |
| ipaddr                          | 1.2.4        |
| io-wait                         | 0.2.1        |
| zlib                            | 2.1.1        |
| resolv                          | 0.2.1        |
| time                            | 0.2.2        |
| open-uri                        | 0.2.0        |
| mutex_m                         | 0.1.1        |
| net-http                        | 0.3.0        |
| net-protocol                    | 0.1.2        |
| ostruct                         | 0.5.2        |
| english                         | 0.7.1        |
| erb                             | 2.2.3        |
| strscan                         | 3.0.1        |
| abbrev                          | 0.1.0        |
| io-console                      | 0.5.11       |
| tempfile                        | 0.1.2        |
| delegate                        | 0.2.0        |
| fileutils                       | 1.6.0        |
| tmpdir                          | 0.1.2        |
| base64                          | 0.1.1        |
| singleton                       | 0.1.1        |
| open3                           | 0.1.1        |
| nkf                             | 0.1.1        |
| prettyprint                     | 0.1.1        |
| pp                              | 0.3.0        |
| find                            | 0.1.1        |
| yaml                            | 0.2.0        |
| psych                           | 4.0.4        |
</details>

*generated on:* **2023-11-01**
</details>

**Comment by chris1058** [comment]
Hello, @Blobbyguy 

It sounds like you're experiencing a frustrating issue with the `add_photos` argument in the simulator. Here are a few steps to troubleshoot and potentially resolve the issue:

Timing Issue: Ensure that the photos are added before the simulator fully starts. This can sometimes be a timing issue.
Manual Check: Verify manually that the photos are indeed in the correct directory before starting the simulator.
Simulator State: Try not erasing the simulator state as it seems to sometimes help. Instead, manually reset if needed.
Update and Reinstall: Ensure you have the latest version of the simulator. Reinstall it if necessary.
Documentation and Support: Double-check any official documentation for updates or known issues. Please contact support for help if the problem continues.

Hopefully, this helps resolve the issue.

Best regards 
chris1058

**Comment by apfelnutzer** [comment]
Same issue here - sometimes it works, most times it doesn't.
